lte/gateway/deploy/agw_install_docker.sh

```diff
@@ -61,6 +85,22 @@ git clone "${GIT_URL}" /opt/magma
 cd /opt/magma || exit
 git checkout "$MAGMA_VERSION"
 
+# changing intefaces name
+sed -i 's/GRUB_CMDLINE_LINUX=""/GRUB_CMDLINE_LINUX="net.ifnames=0 biosdevname=0"/g' /etc/default/grub
+sed -i 's/ens5/eth0/g' /etc/netplan/50-cloud-init.yaml
```

Comment:
I understand and agree this would help in the short term to set the interface name to the same values configured, but ideally we should plan to move away from the traditional naming conventions (eth0 etc.) and move to predictable network interface names.